tests/bud.bats

```diff
@@ -1797,3 +1797,29 @@ load helpers
   buildah rm --all
   buildah rmi --all --force
 }
+
+@test "bud containerfile with tar archive in copy" {
+  # First check to verify cache is uses if the tar file does not change
+  target=copy-archive
+  date>bud/${target}/test; tar cJf - bud/${target}/test > bud/${target}/test.tar.xz
```

Comment:
As a general rule I find `tar cf DESTFILE PATH` more readable than `tar cf - PATH > DESTFILE`, but that's a minor nit and not worth resubmitting for. (If you have a strong preference for the redirection form, I'm willing to hear reasons for it though).